.circleci/config.yml

```
@@ -31,6 +31,11 @@ jobs:
           command: |
             pydocstyle
 
+      - run:
+          name: style-check-type-hints
```

Is this really a style check?
```suggestion
          name: check-type-hints
```

---

I agree -- I was just matching the existing pattern. I renamed the CI job to "pre-checks" so it's all-encompassing, and renamed the individual checks according to their purpose.